### ISSUE TITLE
Tracking issue to add warnings and explanation on Soroban resource limits

### ISSUE THREAD

**Issue by anupsdf** [stellar/stellar-docs#1008]
This [document](https://docs.google.com/document/d/14phzY5DYwpsvYTSPIoh4uOQ9k7na5-m8NxDAky8mao4/edit) during a recent network settings upgrade. There are ideas there to add warning in our dev tooling but we can also added detailed explanation for the Soroban contract/transaction limits in our docs. Creating this tracking issue for Core team and Docs team to collaborate on this.

**Comment by dmkozh** [comment]
I agree that some warnings would help in the tooling, but IIUC this issue is for the docs specifically.

However, I'm not sure what else beyond https://developers.stellar.org/docs/networks/resource-limits-fees should we provide. I can't recall a particular case where this doc wasn't sufficient for evaluating the resources (and again, automated warnings in preflight would indeed be helpful).

**Comment by ElliotFriend** [comment]
Closing: the concrete docs need is covered by `docs/networks/resource-limits-fees.mdx` (per the earlier discussion), and the remaining ideas here are tooling warnings that belong in the CLI/SDK repos rather than docs. If a specific docs gap emerges, a scoped issue will serve better than this tracker.
